ratatui/Cargo.toml

```diff
@@ -15,25 +15,22 @@ edition.workspace = true
 rust-version.workspace = true
 
 [dependencies]
-crossterm = { version = "0.28.1", optional = true }
 document-features = { workspace = true, optional = true }
 indoc = "2"
 instability.workspace = true
 itertools.workspace = true
 palette = { version = "0.7.6", optional = true }
 ratatui-core = { workspace = true }
+ratatui-crossterm = { workspace = true, optional = true }
+ratatui-termion = { workspace = true, optional = true }
```

Comment:
Does it make sense to put this down in the `[target.'cfg(not(windows))'.dependencies]` section?

Reply:
da148bf80..29729ed20